tools/tests/BFTest.sh

```diff
@@ -0,0 +1,89 @@
+check_status()
```

Comment:
Doesn't look like this function is used. Also, you need to put the copyright header in place, and make sure thta is starts with `#!/bin/bash -e`